src/PKSim.Presentation/Presenters/Compounds/CompoundTypeGroupPresenter.cs

```diff
@@ -85,10 +85,11 @@ public IEnumerable<CompoundType> AllCompoundTypes()
 
       public void EditCompoundParameters(IEnumerable<IParameter> compoundParameters)
       {
-         _compoundTypeParameters = compoundParameters.Where(x => string.Equals(x.GroupName, CoreConstants.Groups.COMPOUND_PKA)).ToList();
-         _parameterCache = new PathCache<IParameter>(_entityPathResolver).For(_compoundTypeParameters);
-         _compoundTypeDTO = _dtoMapper.MapFrom(_compoundTypeParameters);
-         _view.BindTo(_compoundTypeDTO.AllTypePKas);
+         var allCompoundPKaParameters = compoundParameters.Where(x => string.Equals(x.GroupName, CoreConstants.Groups.COMPOUND_PKA)).ToList();
+         _parameterCache = new PathCache<IParameter>(_entityPathResolver).For(allCompoundPKaParameters);
+         _compoundTypeDTOs = _dtoMapper.MapFrom(allCompoundPKaParameters);
+         _compoundTypeParameters = _compoundTypeDTOs.SelectMany(x => x.Parameters).ToList();
```

Comment:
These are all the parameters actually editable. we need to keep the list handy so that we can update all ValueOrigin at once

Reply:
I don't see why we have now only EDITABLE parameters here (as opposite to the previous state, where ALL parameters were passed)